CHANGELOG.rst

```diff
@@ -14,6 +14,9 @@ Change Log
 Unreleased
 ~~~~~~~~~~
 
+[3.0.1] - 2021-02-05
```

Comment:
nit: this should probably be a minor version rather than a patch; it adds a pretty hefty feature.